tasks/marketplacesvm-push-disk-images/marketplacesvm-push-disk-images.yaml

```diff
@@ -0,0 +1,143 @@
+---
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: marketplacesvm-push-disk-images
+  labels:
+    app.kubernetes.io/version: "0.1.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task to push disk images to Cloud Marketplaces
+  params:
+    - name: snapshotPath
+      type: string
+      description: |
+        Path to the JSON string of the mapped snapshot spec in the data workspace.
+        It must be processed by the "apply-mapping" task first.
+    - name: cloudMarketplacesSecret
+      type: string
+      description: Env specific secret containing the marketplaces credentials.
+    - name: concurrentLimit
+      type: string
+      description: The maximum number of images to be pulled at once.
+      default: 3
+  workspaces:
+    - name: data
+      description: The workspace where the snapshot spec json file resides
+  steps:
+    - name: pull-and-push-images-to-marketplaces
+      image: quay.io/konflux-ci/release-service-utils:6556e8a6b031c1aad4f0472703fd121a6e1cd45d
+      env:
+        - name: CLOUD_CREDENTIALS
+          valueFrom:
+            secretKeyRef:
+              name: $(params.cloudMarketplacesSecret)
+              key: key
+      script: |
+        #!/usr/bin/env bash
+        set -ex
+
+        # Setup required variables
+        SNAPSHOT_JSON=$(jq -c '.' "$(workspaces.data.path)/$(params.snapshotPath)")
+        STARMAP_MAPPING=$(jq -c '[.components[].starmap[]]' <<< "$SNAPSHOT_JSON")
+        STARMAP_MAPPING_FILE="$(dirname "$(params.snapshotPath)")/starmap.yaml"
+        yq -p json -o yaml <<< "$STARMAP_MAPPING" > "$STARMAP_MAPPING_FILE"
+
```

Comment:
I was trying to pick the file after the task ended. But it seems it deletes the fail when the task ends, as it's stored in `$(dirname "$(params.snapshotPath)")/starmap.yaml`.

My goal on the `mocks` was to write in the `workspaces.data` so it would be preserved but it seems it's gone before

Reply:
basically:
-  `.starmap.yaml` missing after task finishes
- no data stored in worspaces.data

Reply:
I've pushed again using the `workspace.data` within the task, let's see if it works now

Reply:
I mean it doesn't really matter if it works with the path you changed it back to... I can't/won't approve it with that path. It breaks parallelism

Reply:
> My goal on the mocks was to write in the workspaces.data so it would be preserved but it seems it's gone before

The task is writing it. Why are you also writing it in mocks.sh?

Reply:
Use `$(workspaces.data.path)/$(dirname "$(params.snapshotPath)")/starmap.yaml`

Reply:
Updated🤞🏻